recipes/tesseract/all/conanfile.py

```diff
@@ -0,0 +1,111 @@
+import os
+import shutil
+from conans import ConanFile, CMake, tools
+from conans.model.version import Version
+from conans.errors import ConanInvalidConfiguration
+
+
+class TesseractConan(ConanFile):
+    name = "tesseract"
+    description = "Tesseract Open Source OCR Engine"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "ocr", "image", "multimedia", "graphics")
+    license = "Apache-2.0"
+    homepage = "https://github.com/tesseract-ocr/tesseract"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "with_training": [True, False]}
+    default_options = {'shared': False, 'fPIC': True, 'with_training': False}
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+    _cmake = None
+
+    requires = "leptonica/1.79.0"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.remove("fPIC")
+        if self.options.with_training:
+            # do not enforce failure and allow user to build with system cairo, pango, fontconfig
+            self.output.warn("*** Build with training is not yet supported, continue on your own")
+
+    def configure(self):
+        # Exclude old compilers not supported by tesseract
+        compiler_version = Version(str(self.settings.compiler.version))
```

Comment:
```suggestion
        compiler_version = Version(self.settings.compiler.version)
```

Reply:
It fails with `'settings.compiler.version' possible configurations are none`. Looks strange

Reply:
It was another `Version` class imported - `conans.model.version.Version` instead of `tools.Version`.

We should probably fix another [affected recipes](https://github.com/conan-io/conan-center-index/search?q=%22from+conans.model.version+import+Version%22&unscoped_q=%22from+conans.model.version+import+Version%22) .